app/views/planning_applications/validation/description_change_validation_requests/_show.html.erb

```diff
@@ -56,5 +59,20 @@
         </div>
       <% end %>
     <% end %>
+    <% if @validation_request.closed? %>
+      <%= form_with(
+            model: @planning_application,
+            url: validate_planning_application_validation_description_changes_path(
+              @planning_application
+            )
+          ) do |form| %>
+          <%= form.hidden_field(:return_to, value: @back_path) %>
+          <%= form.hidden_field(:valid_description, value: true) %>
+          <div class="govuk-button-group">
+            <%= form.govuk_submit("Save and mark as complete") %>
+            <%= govuk_link_to(t(".back"), @back_path, class: "govuk-button govuk-button--secondary") %>
+          </div>
```

Comment:
<%= form.govuk_submit("Save and mark as complete") do %>
  <%= govuk_button_link_to("back", @back_path, secondary: true) %>
<% end %>